src/hooks/transactions/batch/tracker/useBatchTransactionsTracker.ts

```diff
@@ -38,14 +40,16 @@ export const useBatchTransactionsTracker = ({
 
   const checkAllBatches = async () => {
     for (const [sessionId, session] of signedTransactionsArray) {
-      if (
-        session.status !== TransactionBatchStatusesEnum.sent &&
-        session.status !== TransactionBatchStatusesEnum.signed
-      ) {
+      const batchId = `${sessionId}-${address}`;
```

Comment:
best to use a constructor function here

Reply:
I don't see the necessity of having a dedicated function for a string concatenation